src/extension.ts

```diff
@@ -681,13 +682,13 @@ function registerCommands(context: VSCode.ExtensionContext) {
     })
   );
 
-  context.subscriptions.push(
```

Comment:
The commands have `        "enablement": "sonarqube.mcpServerSupportedIDE"` in the `package.json`. This check should not be needed